blueprintapi/Environment.py

```diff
@@ -13,7 +13,7 @@ def __init__(self):
             "django-insecure-_o$0y5g@1*uyrw0!3(0%wdv-ds5wp26yp*bko+q#y4b&y!50%6"
         )
         hosts = ["localhost", "127.0.0.1"]
-        env_hosts = os.environ.get("ALLOWED_HOSTS")
+        env_hosts = os.environ.get("ALLOWED_HOSTS", "localhost")
```

Comment:
@azharem there were a few environment variables that I thought could use some reasonable defaults to make life easier when developing locally.

Reply:
@jwhite3 The default "localhost" is already part of the Allowed_host. Look at line 15.

Reply:
Yes, but before, `ALLOWED_HOSTS` was effectively not required, while now it is. So it has to be set to something since it's used in constructing `CSRF_TRUSTED_ORIGINS`, and I figured that "localhost" would be a reasonable default even though it meant that `ALLOWED_HOSTS` had a duplicate entry when the default is used.

Reply:
In that case I'd suggest to remove "localhost" from line 15 to make it like this: 

> hosts = ["127.0.0.1"]

Reply:
I worried about the assumption that `ALLOWED_HOSTS` wouldn't be set to something else, which would mean that "localhost" would be missing from the allowed hosts django setting and could cause issues for local development. 

I compromised and switched to using a `set` to avoid duplicate entries in the allowed hosts list (which is probably a good idea anyways until we can switch to using the Django-environ plugin).